explorer/app/apis/catalog/anvil-catalog/common/transformers.ts

```diff
@@ -83,6 +119,31 @@ export const getStudyDesigns = (
   anvilCatalogEntity: AnVILCatalogEntity
 ): string[] => anvilCatalogEntity.studyDesigns ?? [];
 
+/**
+ * Maps dataset-related information, included formatted display text from API response.
+ * @param anvilCatalogStudy - the study to use to extract details.
+ * @returns data summaries key-value pairs of data summary label and corresponding value.
+ */
+export function getStudyDetails(
+  anvilCatalogStudy: AnVILCatalogStudy
+): KeyValues | undefined {
```

Comment:
The return value is KeyValues. The `details` map will always have "Consortium" and "PHSID" as a key for example.